Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -150,6 +150,11 @@ open class ChatMessageContentView: _View, ThemeProvider, UITextViewDelegate {
         return appearance.colorPalette.text
     }
 
+    /// The character separator of the "Edited" label.
+    open var editedLabelSeparator: String {
```

Comment:
Not sure if we should expose this - maybe better to just replace the whole view if needed (which we already have). Just a thought - leave it up to you.

Reply:
I think it is a nice to have, I'll prefer to keep it just in case 🤔